config/config.yaml

```diff
@@ -5,15 +5,21 @@ hash_length: 7
 
 # Specify the container framework used by each PRM wrapper. Valid options include:
 # - docker (default if not specified)
-# - singularity -- Also known as apptainer, useful in HPC/HTC environments where docker isn't allowed
-# - dsub -- experimental with limited support, used for running on Google Cloud with the All of Us cloud environment.
-#   - There is no support for other environments at the moment.
+# - singularity OR apptainer -- Apptainer (formerly Singularity) is useful in HPC/HTC environments where docker isn't allowed
+# - dsub -- experimental with limited support, used for running on Google Cloud
 container_framework: docker
 
-# Only used if container_framework is set to singularity, this will unpack the singularity containers
+# Enabling profiling adds a file called 'usage-profile.tsv' to the output directory of each algorithm.
+# The contents of this file describe the CPU utilization and peak memory consumption of the algorithm
+# as seen by its runtime container.
+# NOTE: Profiling is currently supported only when the container framework is set to apptainer/singularity
+#       and when the host system supports the 'cgroup' filesystem.
```

Comment:
@jhiemstrawisc would this be a good place to document the requirement for HTCondor >= 24.8.0?

We should move documentation of the config file to readthedocs. In the meantime `config/config.yaml` is our placeholder to gather that information.